Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -26,7 +26,7 @@
 @interface UserProfile : TSYapDatabaseObject
 
 @property (atomic, readonly) NSString *recipientId;
-@property (atomic, nullable) OWSAES128Key *profileKey;
+@property (atomic, nullable) OWSAES256Key *profileKey;
```

Comment:
Most likely yes, though I didn't have a problem, probably because I had reinstalled my development devices. I don't want to complicate the production code, but if you'd like I could add a debug action that regenerates the local profile key.